pupil_src/shared_modules/eye_movement/eye_movement_detector_offline.py

```diff
@@ -91,14 +95,19 @@ def trigger_recalculate(self):
     def seek_to_timestamp(self, timestamp):
         self.notify_all({"subject": "seek_control.should_seek", "timestamp": timestamp})
 
+    def on_task_started(self):
+        self.menu_content.update_error_text("")
+
     def on_task_progress(self, progress: float):
         self.menu_content.update_progress(progress)
 
     def on_task_status(self, status: str):
         self.menu_content.update_status(status)
 
     def on_task_exception(self, exception: Exception):
-        raise exception
+        error_message = f"{exception}"
+        logger.exception(error_message)
```

Comment:
```suggestion
        logger.error(error_message)
```
Even though `logger.exception()` is appropriate here, I do not know if the zmq log handler is able to handle the included exception correctly.